src/disks.py

```diff
@@ -75,15 +62,15 @@ def list(with_info=False, human_readable=False):
     if not with_info:
         return [name for name, _ in _query_udisks()]
 
-    result = {}
+    result = []
 
     for name, drive in _query_udisks():
-        result[name] = _disk_infos(name, drive, human_readable)
+        result.append(_disk_infos(name, drive, human_readable))
 
     if human_readable and not result:
         return "No external media found"
 
-    return result
+    return {"disks": result}
```

Comment:
Should `result` be sorted some way?

Reply:
I guess it could be better to have sda before sdb... So alphanumeric sort ?

Reply:
[Udisks2 provides a `SortKey` property](https://storaged.org/doc/udisks2-api/latest/gdbus-org.freedesktop.UDisks2.Drive.html#gdbus-property-org-freedesktop-UDisks2-Drive.SortKey). I don't know how it is computed though. I can do a `sda`/`sdb`, etc. sorting. But do we have the garantuee that a disk is always present in `/dev`?

Reply:
Ok, done.